.github/workflows/refapp-3x-patient-registration.yml

```diff
@@ -14,11 +14,18 @@ jobs:
       run:
         working-directory: ./qaframework-bdd-tests
     steps:
-      - uses: actions/checkout@v2
+      - name: Checkout qaframework
+        uses: actions/checkout@master
+        with:
+          repository: ${{github.repository}}
```

Comment:
You do not need to add this section to the workflow, because `repository: ${{github.repository}}` contains a default value.